### DISCUSSION TITLE
Incorrect displaying in ConEmu

### DISCUSSION THREAD

**Discussion by deNoi5e** [NvChad/NvChad#2231] [General]
Hey, everybody. Please tell me if anyone has encountered incorrect display of colors in ConEmu (when using both PowerShell and CMD). The background becomes bright purple. I tried the xterm-256color solution - no results.
<details>
  <summary>Screenshot</summary>
  
 ![image](https://github.com/NvChad/NvChad/assets/9005584/d8372e6a-e497-4106-8f43-d2cd10a08785)
  
</details>

**Comment by antoineco** [comment]
It is likely that ConEmu doesn't have 24-bit True Color enabled by default. Luckily, this seems to be configurable:

> Options must be turned on
>  - **TrueMod (24bit color) support** on [Colors](https://conemu.github.io/en/Settings.html#Colors) page
>  - [...]
>
> -- https://conemu.github.io/en/Xterm256Colors.html

The `termguicolor` (Neo)vim setting is [set to `true`](https://github.com/NvChad/NvChad/blob/f18488d4/lua/core/init.lua#L41) in NvChad, because its UI components depend on it.
